crates/rattler_lock/src/conda.rs

```diff
@@ -0,0 +1,215 @@
+use crate::{
+    LockedDependency, LockedDependencyKind, PackageHashes,
+    PackageHashes::{Md5, Md5Sha256, Sha256},
+};
+use rattler_conda_types::{
+    InvalidPackageNameError, NoArchType, PackageName, PackageRecord, ParseMatchSpecError,
+    ParseVersionError, RepoDataRecord,
+};
+use serde::{Deserialize, Serialize};
+use serde_with::{serde_as, skip_serializing_none, OneOrMany};
+use url::Url;
+
+/// A locked conda dependency. This represents a [`rattler_conda_types::RepoDataRecord`].
+#[serde_as]
+#[skip_serializing_none]
+#[derive(Serialize, Deserialize, Eq, PartialEq, Clone, Debug)]
+pub struct CondaLockedDependency {
+    /// What are its own dependencies mapping name to version constraint
+    #[serde(default)]
+    #[serde_as(deserialize_as = "crate::utils::serde::MatchSpecMapOrVec")]
+    pub dependencies: Vec<String>,
+    /// URL to find it at
+    pub url: Url,
+    /// Hashes of the package
+    pub hash: PackageHashes,
+    /// ???
```

Comment:
Do we know more about this field?

Reply:
No, I copied this from the previous implementation